Bot.js

```diff
@@ -4,59 +4,73 @@ const client = new Discord.Client();
 
 //Constant for commands file
 const {commands} = require('./src/');
-
-var commandChannels = [];
+const {prefix, token} = require('./config/Bot.json')
+const blacklist = [];
 
 //On ready, output logged in message
-client.on('ready', () =>
-{
-  console.log(`Logged in as ${client.user.tag}`);
-
-  //Gets command channels where you put bot commands in
-  commandChannels = Commands.getCommandChannels(client);
+client.once('ready', () => {
+	console.log("Logged in as "+client.user.username+"#"+client.user.discriminator);
+	console.log("Guild Count: "+client.guilds.array().length)
```

Comment:
If I'm running on a server, why would I want to output log to console? I think logging isn't a bad idea; however, I think it's more beneficial to log to a file is it not?

Reply:
You can leave this as is for now, at least add a todo comment for logging or having the option to run the bot without debug maybe?